addons/survey/static/src/js/survey_form.js

```diff
@@ -254,6 +257,34 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
         }
     },
 
+    /**
+     * Called when an image on an answer in muti-answers question is clicked.
+     * Starts a widget opening a dialog to display the image with a bigger size.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onChoiceImgClick: function (ev) {
+        ev.preventDefault();
+        var $img = $(ev.currentTarget).find('img');
+
+        var $modal = $(qweb.render('survey.survey_question_choice_image_zoom', {
+            sourceImage: $img.attr('src'),
+        }));
+        $modal.modal({
+            keyboard: true,
+            backdrop: true,
```

Comment:
The backdrop is indeed required imho, but cannot be accessed as it is.

Furthermore, I would suggest that clicking anywhere (on the picture or around it) should close the modal in this case, as there is nothing else to do.

Reply:
What I did is to remove the backdrop (diff -2 lines) and just allow the user to close the modal by clicking anywhere.